basxconnect/core/migrations/0009_auto_20210522_1817.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.1.7 on 2021-05-22 11:17
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0008_auto_20210323_1456"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="relationship",
+            name="person_a",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="relationships_to",
```

Comment:
it it not the other way around? that `person_a` should become `relationship_from`?

Reply:
That was my initial thought too. But looking at it closely showed me that it has to be this way, otherwise the semantics are wrong when looking at entries in the table. Because the related_name determines the name of the attribute on the person_a object.

I feel like I already changed this system one or two times, forth and back :D
For clarification I drew a small diagram, hope we get it correct this time.

![signal-2021-05-24-160827](https://user-images.githubusercontent.com/3115209/119325728-6089b880-bcab-11eb-81c5-09885eaa1fe4.jpeg)

Reply:
Hey look, hand-drawn sketches really give some character to a the all-markdowny GitHub :D

Reply:
Yes, the sketch makes sense and looks correct! :)